helm/values.yaml

```diff
@@ -91,3 +92,20 @@ logConfig:
   appender:
     rolling:
       enabled: false
+
+kafka-topic-creator:
+  enabled: true
+  jobName: config-service-kafka-topics-creator
+  helmHook: pre-install,pre-upgrade
+  kafka:
+    topics:
+      config-change-events:
+        replicationFactor: 3
+        partitions: 8
+        configs:
+          retention.bytes: 1073741824 # default = -1
```

Comment:
curious about what is this value (in MB?) and also why this is the right value and not for example the half of it. It is good to have this explanation in the code so next person willing to change it has the right context.